pkgs/development/python-modules/biothings-client/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pytestCheckHook,
+  requests,
+}:
+buildPythonPackage rec {
+  pname = "biothings-client";
+  version = "0.3.1";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
PyPI tarball seems to have no tests. Could you fetch the source from GitHub instead?
https://github.com/NixOS/nixpkgs/pull/344340/checks?check_run_id=30805512453

Reply:
I rigged this up and tested it locally, but it appears as though the large majority of their tests make network calls to pull in data over the network (which I gather is [not allowed in a derivation](https://discourse.nixos.org/t/using-dns-in-a-nix-derivation-temporary-failure-in-name-resolution/20543/4). Is there a standard way to handle that?

Reply:
You can use the attributes `disabledTests` and `disabledTestPaths`.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook

Reply:
I listed out the non-network tests that pass when I build locally with a comment indicating the rest require network access.